src/lib/y2storage/boot_requirements_strategies/base.rb

```diff
@@ -117,6 +122,17 @@ def boot_partition_needed?
         false
       end
 
+      def too_small_boot?
+        # for other partitions it is not needed as packager check disk usage, but /boot is special
+        # as it contain initrd that is generated and also bootloader code.
+        filesystem = devicegraph.filesystems.find { |f| f.mount_point == "/boot" }
+        return false unless filesystem
+
+        available_size = filesystem.blk_devices.map(&:size).reduce(:+)
+        # FIXME: it does not count fs size and other and it is just hard coded.
+        return available_size < DiskSize.MiB(100) # for number see below minimal size for boot volume
```

Comment:
Instead of a comment with "see below minimal size"... why don't you really use that minimal size?
```ruby
available_size < boot_volume.min_size
```

By the way, `return` is surplus and anti-ruby :wink:

Reply:
makes sense. BTW about return, usually rubocop reminds me that :)